Proiect/src/CustomerService.java

```diff
@@ -0,0 +1,65 @@
+import java.util.Map;
+import java.util.Scanner;
+import java.util.TreeMap;
+
+public class CustomerService extends UserService
+{
+    private Map<String ,Customer> customer_list = new TreeMap<String ,Customer>();

```

Comment:
use just ` new TreeMap<>();`

Reply:
also, rename all fields to be camelCase, no _  used in Java practice for method names/variable names